#mode0: loc mode1: mapping
slam_mode_switch: 0
# 1 for debug (pub to rviz),0 for not debug
if_debug: 0

gnss_align_threshold: 0.02
gnss_pop_threshold: 0.0

# ground filter
use_ground_filter: false
min_grid_pt_num: 8
#max_ground_height: DBL_MAX #1111111111111
grid_resolution: 1.5
distance_weight_downsampling_method: 0
standard_distance: 15.0
nonground_random_down_rate: 1
intensity_thre: 5000.0
apply_grid_wise_outlier_filter: false
outlier_std_scale: 3.0
reliable_neighbor_grid_num_thre: 0
ground_random_down_rate: 1
neighbor_height_diff: 25.0
max_height_difference: 0.3
estimate_ground_normal_method: 3
normal_estimation_radius: 2.0
fixed_num_downsampling: false
ground_random_down_down_rate: 2

# unground pts classify
use_unground_pts_classify: false
neighbor_searching_radius: 1.0
neighbor_k: 30
neigh_k_min: 8
pca_down_rate: 1
edge_thre: 0.65
planar_thre: 0.65
edge_thre_down: 0.75
planar_thre_down: 0.75
extract_vertex_points_method: 2
curvature_thre: 0.12
vertex_curvature_non_max_radius: 1.5
linear_vertical_sin_high_thre: 0.94
linear_vertical_sin_low_thre: 0.17
planar_vertical_sin_high_thre: 0.98
planar_vertical_sin_low_thre: 0.34
pillar_down_fixed_num: 200
facade_down_fixed_num: 800
beam_down_fixed_num: 200
roof_down_fixed_num: 100
unground_down_fixed_num: 20000
#beam_height_max: FLT_MAX
#roof_height_min: -FLT_MAX
feature_pts_ratio_guess: 0.3
sharpen_with_nms: true
use_distance_adaptive_pca: false